__test__/lib/favorite-mock.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const Favorite = require('../../model/favorite.js');
+
+let create = (profile, charity) => {
```

Comment:
this mocks could be more expressave if the generated a profile and charity, instead of expecting them through paramiters